src/CommunityToolkit.Maui.Core/readme.txt

```diff
@@ -0,0 +1,25 @@
+﻿.NET MAUI Community Toolkit
+
+## Initializing
+
+In order to use the .NET MAUI Community Toolkit you need to call the extension method in your `MauiProgram.cs` file as follows:
+
+```csharp
+public static MauiApp CreateMauiApp()
+{
+    var builder = MauiApp.CreateBuilder();
+
+    // Initialise the toolkit
+	builder.UseMauiApp<App>().UseMauiCommunityToolkitCore();
+
+    // the rest of your logic...
+}
+```
+
+## Further information
+
+For more information please visit:
+
+- Our documentation site: https://docs.microsoft.com/dotnet/communitytoolkit/maui
```

Comment:
This site is not currently live. Do we need a note in here making this clear?

Reply:
We can wait to merge this PR until the docs website is live.

I think Dave is close to getting it working 🤞

Reply:
We will likely ship without given this still adds value